test/unit/ngsiService/active-devices-test.js

```diff
@@ -247,6 +247,60 @@ describe('Active attributes test', function() {
         });
     });
 
+    describe('When the IoT Agent receives new information, the timestamp flag is on' +
+    'and timezone is defined', function() {
+        var modifiedValues;
+
+        beforeEach(function(done) {
+            var time = new Date(1438760101468);
```

Comment:
It would be useful a hint to which date this integer corresponds (in a commented line just above the line, for instance). The same for other possible cases in this PR.

Reply:
Fixed with 0f341d164437b4f2b9876ba8acf56d25afa5cefa